code-challenges.js

```diff
@@ -7,43 +7,112 @@
+//42 is below boiling point
+//350 is above boiling point
+//212 is at boiling point
 
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
+//This code combines two assigned arrays and returns the total length. 
+//The arrays will be named myNumbers1 and myNumbers2, and will contain the same amount of numbers. 
+
 const myNumbers1 = [3, 7, 0, 36, -9]
 const myNumbers2 = [8, -7, 42, 9, 13]
 
+//Console.log will be used to display the output with concatenation and length methods implemented. 
+console.log(myNumbers1.concat(myNumbers2).length)
+//The expected output is: 10 
 
-// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ovarB"
 
+// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "2202 ovarB"
+
+//This code reverses the letters of the given string, "Bravo 2022". The string will be contained in the variable called currentCohort. 
 const currentCohort = "Bravo 2022"
+//Console.log will be used to display the output with split and join methods implemented. 
+console.log(currentCohort.split("").reverse().join(""))
+//The expected output is: 2022 ovarB
 
 
 // --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
 
-const myArray = [13, 34, 5, 10, 27, 42]
+//This JavaScript statement evaluates whether each number in a given array is even or odd. 
+//The array will be named myArray. 
+  const myArray = [13, 34, 5, 10, 27, 42]
+//An if/else statement will be used to determine if the numbers are even or odd. 
+ for(let i = 0; i < myArray.length; i ++){
+     if (myArray[i] % 2 == 0){
+      console.log ("even")
+  }else{
+      console.log ("odd")
+  }
+  }
+// //The expected output is: 
+// odd 
+// even 
+// odd 
+// even 
+// odd 
+// even
 
 
 // --------------------5) Create a function that takes in a set of any two numbers and subtracts the smaller number from the larger number using the sets of test variables provided below. Expected output: 42, 3
 
+//This function takes in a set of any two numbers and subtracts the smaller number from the larger number, giving the difference. Two sets of variables are provided, with set one including number1 and number2, and set two including number3 and number4. 
 // Set one:
-const number1 = 58
-const number2 = 100
+ const number1 = 58
+  const number2 = 100
 
 // Set two:
-const number3 = 27
-const number4 = 24
+ const number3 = 27
+  const number4 = 24
+
+// //The function will be called subtractNumbers. 
+ const subtractNumbers = (number1,number2,number3,number4) => {
```

Comment:
Logic looks great!  You only need to have 2 params defined in your parenthesis on line 104 since you only have 2 arguments passed into your function.  Oftentimes, I will call my param something that is different than the variable name to avoid confusion.  In this case, even naming it something like (num1, num2) is descriptive enough but different from your defined variables.